lib/commands/signout.js

```diff
@@ -0,0 +1,44 @@
+const { SignOut } = require('../messages/flow');
+
+const SignedParams = require('../signed-params');
+const getProtocolAndHost = require('../get-protocol-and-host');
+const getRepositories = require('../github/get-repositories');
+const { Subscription } = require('../models');
+const ReEnableMultipleSubscriptions = require('../messages/flow/re-enable-multiple-subscriptions');
+
+module.exports = async (req, res) => {
+  const {
+    slackUser,
+    slackWorkspace,
+    command,
+    robot,
+  } = res.locals;
+
+  const deletedSubscriptions = await slackUser.signout();
+
+  if (deletedSubscriptions) {
+    const subscriptionsByChannel = Subscription.groupByChannel(deletedSubscriptions);
+
+    await Promise.all(Object.keys(subscriptionsByChannel).map(async (channel) => {
+      const subscriptions = subscriptionsByChannel[channel];
+      const repositories = await getRepositories(subscriptions, robot);
+      if (repositories) {
+        const attachment = new ReEnableMultipleSubscriptions(repositories, slackUser.slackId, 'disconnected their GitHub account').getAttachment();
+        return slackWorkspace.client.chat.postMessage({
+          channel,
+          attachments: [attachment],
+        });
+      }
+    }));
```

Comment:
Extra context: I plan to refactor this code, so that it will be used both by `/github signout` AND the code for the "token revoked" event. That event will probably lead to this code path being called much more frequently: https://github.com/integrations/slack/issues/446

In a way this PR was just a set up for this follow up one